src/index.tsx

```diff
@@ -21,6 +22,8 @@ if ('serviceWorker' in navigator) {
   ErrorPage = null
 }
 
+void ensureSwScope()
```

Comment:
we are doing this in index.tsx to ensure the redirect happens as quickly as possible